contrib/mac-install-bundle.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+BUNDLE="bundles/$(cat VERSION)"
+BUNDLE_PATH="$PWD/$BUNDLE"
+DATABASE=~/Library/Containers/com.docker.docker/Data/database
+
+[ -d "$DATABASE" ] || (echo "Docker for Mac must be installed for this script" && exit 1)
+
+if [ "$1" = "undo" ]
+then
+	git -C "$DATABASE" reset --hard >/dev/null
+	[ -f $DATABASE/com.docker.driver.amd64-linux/bundle ] || (echo "bundle not set" && exit 1)
+	git -C "$DATABASE" rm $DATABASE/com.docker.driver.amd64-linux/bundle
+	git -C "$DATABASE" commit -m "remove bundle"
+
+	echo "Bundle removed. Using dev versions may cause issues, a reset to factory defaults is recommended."
+	exit 0
+fi
+
+[ -d "$BUNDLE" ] || (echo "cannot find bundle $BUNDLE" && exit 1)
+
+[ -f "$BUNDLE/binary-client/docker" ] || (echo "you need to build binaries first" && exit 1)
+
+git -C "$DATABASE" reset --hard >/dev/null
+echo "$BUNDLE_PATH" > $DATABASE/com.docker.driver.amd64-linux/bundle
+git -C "$DATABASE" add $DATABASE/com.docker.driver.amd64-linux/bundle
+git -C "$DATABASE" commit -m "update bundle to $BUNDLE_PATH"
+
+echo "Bundle installed. Restart Docker to use. To uninstall, reset Docker to factory defaults."
```

Comment:
Given that this script only updates the *server* side, do we need to mention that people need to take care of the _client_ themselves (and where to install them, so that the "stock" client symlinks are not overwritten?)